libraries/math/tests/instruction_count.rs

```diff
@@ -147,6 +147,38 @@ async fn test_f32_divide() {
     banks_client.process_transaction(transaction).await.unwrap();
 }
 
+#[tokio::test]
+async fn test_f32_exponentiate() {
+    let pc = ProgramTest::new("spl_math", id(), processor!(process_instruction));
+
+    //pc.set_compute_max_units();
```

Comment:
Please set this! It'll be cool to see how the number evolves, and it'll help us catch any undesired changes in computation numbers as new compilers are released.